<launch>
    <group ns="/bat_swarm_red">
        <group ns="bat_1">
            <node pkg="bat_algo" type="go_to_point.py" name="metaHeuristicPlanner" output="screen" />
            <!-- <node pkg="bat_algo" type="go_to_point_combined.py" name="go_to_point" output="screen" /> -->
        </group>
        <group ns="bat_2">
            <node pkg="bat_algo" type="go_to_point.py" name="metaHeuristicPlanner" output="screen" />
            <!-- <node pkg="bat_algo" type="go_to_point_combined.py" name="go_to_point" output="screen" /> -->
        </group>
        <group ns="bat_3">
            <!-- <node pkg="bat_algo" type="go_to_point_combined.py" name="go_to_point" output="screen" /> -->
            <node pkg="bat_algo" type="go_to_point.py" name="metaHeuristicPlanner" output="screen" />
        </group>
        <group ns="bat_4">
            <node pkg="bat_algo" type="go_to_point.py" name="metaHeuristicPlanner" output="screen" />
            <!-- <node pkg="bat_algo" type="go_to_point_combined.py" name="go_to_point" output="screen" /> -->
        </group>
        <group ns="bat_5">
            <node pkg="bat_algo" type="go_to_point.py" name="metaHeuristicPlanner" output="screen" />
            <!-- <node pkg="bat_algo" type="go_to_point_combined.py" name="go_to_point" output="screen" /> -->
        </group>
    </group>
    <group ns="/bat_swarm_green">
        <group ns="bat_6">
            <node pkg="bat_algo" type="go_to_point.py" name="metaHeuristicPlanner" output="screen" />
            <!-- <node pkg="bat_algo" type="go_to_point_combined.py" name="go_to_point" output="screen" /> -->
        </group>
        <group ns="bat_7">
            <node pkg="bat_algo" type="go_to_point.py" name="metaHeuristicPlanner" output="screen" />
            <!-- <node pkg="bat_algo" type="go_to_point_combined.py" name="go_to_point" output="screen" /> -->
        </group>
        <group ns="bat_8">
            <!-- <node pkg="bat_algo" type="go_to_point_combined.py" name="go_to_point" output="screen" /> -->
            <node pkg="bat_algo" type="go_to_point.py" name="metaHeuristicPlanner" output="screen" />
        </group>
        <group ns="bat_9">
            <node pkg="bat_algo" type="go_to_point.py" name="metaHeuristicPlanner" output="screen" />
            <!-- <node pkg="bat_algo" type="go_to_point_combined.py" name="go_to_point" output="screen" /> -->
        </group>
        <group ns="bat_10">
            <node pkg="bat_algo" type="go_to_point.py" name="metaHeuristicPlanner" output="screen" />
            <!-- <node pkg="bat_algo" type="go_to_point_combined.py" name="go_to_point" output="screen" /> -->
        </group>
    </group>
    <group ns="/bat_swarm_blue">
        <group ns="bat_11">
            <node pkg="bat_algo" type="go_to_point.py" name="metaHeuristicPlanner" output="screen" />
            <!-- <node pkg="bat_algo" type="go_to_point_combined.py" name="go_to_point" output="screen" /> -->
        </group>
        <group ns="bat_12">
            <node pkg="bat_algo" type="go_to_point.py" name="metaHeuristicPlanner" output="screen" />
            <!-- <node pkg="bat_algo" type="go_to_point_combined.py" name="go_to_point" output="screen" /> -->
        </group>
        <group ns="bat_13">
            <!-- <node pkg="bat_algo" type="go_to_point_combined.py" name="go_to_point" output="screen" /> -->
            <node pkg="bat_algo" type="go_to_point.py" name="metaHeuristicPlanner" output="screen" />
        </group>
        <group ns="bat_14">
            <node pkg="bat_algo" type="go_to_point.py" name="metaHeuristicPlanner" output="screen" />
            <!-- <node pkg="bat_algo" type="go_to_point_combined.py" name="go_to_point" output="screen" /> -->
        </group>
        <group ns="bat_15">
            <node pkg="bat_algo" type="go_to_point.py" name="metaHeuristicPlanner" output="screen" />
            <!-- <node pkg="bat_algo" type="go_to_point_combined.py" name="go_to_point" output="screen" /> -->
        </group>
    </group>
</launch>